assets/static/js/dataset-submission.js

```diff
@@ -228,6 +230,34 @@ $(function() {
         saveDatasetSubmission(true);
     });
 
+    function getCookie(name) {
+        let cookies = document.cookie.split(';');
+        for(let i=0 ; i < cookies.length ; ++i) {
+            let pair = cookies[i].trim().split('=');
+            if(pair[0] === name)
+                return pair[1];
+        }
+        return null;
+    }
+
+    function populateFolderDropDownList() {
+        let dropdown = $('#datasetFilePath');
+
+        dropdown.empty();
+
+        dropdown.append('<option selected="true" disabled>Choose Folder</option>');
+        dropdown.prop('selectedIndex', 0);
+
+        const url = Routing.generate("pelagos_api_get_folder_list_dataset_submission", { "user" : getCookie("GRIIDC_USERNAME")} );
```

Comment:
Should we try and minimize (or phase out) the use of this cookie? I thought it was only for drupal, which at some point may be moved to another system altogether.

Reply:
This cookie won't be needed, the code is being changed elsewhere for that.

Reply:
I'll remove this, not required. I can get the user in the Controller like how @mickel1138  mentioned